package.json

```diff
@@ -58,13 +65,31 @@
     "eslint-plugin-react-hooks": "^1.6.0",
     "eslint-plugin-standard": "^4.0.0",
     "jest": "^26.4.2",
+    "microbundle": "^0.12.4",
     "prettier": "^1.17.1",
     "react": "16.8.3",
     "react-dom": "16.8.3",
+    "size-limit": "^4.6.2",
     "ts-jest": "^26.3.0",
     "typescript": "^3.5.1"
   },
   "resolutions": {
     "kind-of": "6.0.3"
-  }
+  },
+  "size-limit": [
+    {
+      "path": "dist/index.module.js",
+      "limit": "1.5 KB"
+    },
+    {
+      "path": "dist/index.module.js",
+      "import": "{ useDebounce }",
+      "limit": "1 KB"
+    },
+    {
+      "path": "dist/index.module.js",
+      "import": "{ useDebouncedCallback }",
+      "limit": "1 KB"
+    }
+  ]
 }
```

Comment:
yarn.lock  is quite a good thing even for open-source projects. 

We get used to working with SemVer standard of npm packages, even though not all packages follow it. 

A fixed version for each package could guarantee that our project will work tomorrow, next month, or even next year without digging into dependencies.